src/tests/multihost/alltests/test_backtrace.py

```diff
@@ -103,7 +103,7 @@ def test_0001_bz2021196(self, multihost, backupsssdconf):
             decode('utf-8')
         print(f'\n{logfile}\n+===++++++===+\n{log_str2}\n')
         # Check the backtrace is dumped first time and no backtrace is skipped
-        assert pattern.search(log_str1) and not pattern2.search(log_str1)
```

Comment:
This does not look correct. You are expecting now that "skipping repetitive backtrace " is present but is should be there after the second call of getent passwd fakeuser, you are truncating logs in the meantime which might affect the functionality. You should count the number of occurrences of "skipping repetitive backtrace" without truncating logs between and comparte it after second "getenmt passwd".
```
       1. Modify sssd.conf with a typo in the ldap_uri
          2. Remove debug_level from domain section
          3. Start sssd
          4. Truncate domain logs
          5. Lookup any user
          6. Check logs generate a backtrace, count number of "skipping repetitive backtrace"
          7. Lookup same user again
          8. Check logs dont have repeated backtrace, the count of "skipping repetitive backtrace" is bigget than in step 6.
```

Reply:
This test was written by @shridhargadekar , i think i will drop this change and ask him to revisit this test and do the modification as per new requirement